### PR TITLE
Added Collab section & Navbar links

### PR DESCRIPTION
**Pull request by bparker12**
Please review the changes made to index.html file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,32 +1,59 @@
...
+      </nav>
+    </div>
+    <div>
+      <!-- This section includes all the collabs for beethoven.  Intent will be to make it a flex box underneath the image with the other 2 sections 
+        that are going to be included in the html file.  (other 2 sections include Aliases and Country of Residence) -->
+      <h2>Collaborations""</h2>
+      <p>
+        During the time period of Beethoven's life, collaborations with other
+        artists were not commonly practiced. Beethoven did have a couple of
+        "colaborations" with other artists though. They were:
```
Comment (DannyBarker) [inline index.html]: spelled "collaborations" wrong

**Review by MDzola — APPROVED** [review]
👍

File: index.html (hunk 1 above)
Comment (bparker12) [reply index.html]: updated

**Review by DannyBarker — APPROVED** [review]

**Comment by DannyBarker** [comment]
👍

**Pull request merged**